xbmc/pvr/recordings/PVRRecording.cpp

```diff
@@ -192,7 +193,8 @@ bool CPVRRecording::operator==(const CPVRRecording& right) const
           m_parentalRating == right.m_parentalRating &&
           m_parentalRatingCode == right.m_parentalRatingCode &&
           m_parentalRatingIcon == right.m_parentalRatingIcon &&
-          m_parentalRatingSource == right.m_parentalRatingSource);
+          m_parentalRatingSource == right.m_parentalRatingSource) &&
+             m_episodePartNumber == right.m_episodePartNumber;
```

Comment:
Wrong indentation. Jenkins will tell you.

Reply:
I didn't have this originally but the change came from this diff that jenkins provided on my first submit.

```
diff --git a/xbmc/pvr/recordings/PVRRecording.cpp b/xbmc/pvr/recordings/PVRRecording.cpp
index 3c6747b95b..a44c85df33 100644
--- a/xbmc/pvr/recordings/PVRRecording.cpp
+++ b/xbmc/pvr/recordings/PVRRecording.cpp
@@ -194,7 +194,7 @@ bool CPVRRecording::operator==(const CPVRRecording& right) const
           m_parentalRatingCode == right.m_parentalRatingCode &&
           m_parentalRatingIcon == right.m_parentalRatingIcon &&
           m_parentalRatingSource == right.m_parentalRatingSource) &&
-          m_episodePartNumber == right.m_episodePartNumber;
+             m_episodePartNumber == right.m_episodePartNumber;
 }
 
 bool CPVRRecording::operator!=(const CPVRRecording& right) const
diff --git a/xbmc/pvr/recordings/PVRRecording.h b/xbmc/pvr/recordings/PVRRecording.h
index aeab432a24..e0ebeee57b 100644
--- a/xbmc/pvr/recordings/PVRRecording.h
+++ b/xbmc/pvr/recordings/PVRRecording.h
@@ -9,8 +9,8 @@
 #pragma once
 
 #include "XBDateTime.h"
-#include "addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr/pvr_providers.h"
 #include "addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr/pvr_epg.h"
+#include "addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr/pvr_providers.h"
 #include "addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr/pvr_recordings.h"
 #include "pvr/PVRCachedImage.h"
 #include "threads/CriticalSection.h"
```

Reply:
OK right you are, sorry I should have looked more carefully when the diff didn't make sense.